t/14-isotovideo.t

```diff
@@ -30,9 +30,11 @@ sub isotovideo (%args) {
     $args{default_opts} //= 'backend=null';
     $args{opts} //= '';
     $args{exit_code} //= 1;
-    my @cmd = ($^X, "$toplevel_dir/isotovideo", '-d', $args{default_opts}, split(' ', $args{opts}));
+    chdir "$Bin/..";
+    my @cmd = ($^X, "$toplevel_dir/isotovideo", '--workdir', $pool_dir, '-d', $args{default_opts}, split(' ', $args{opts}));
+    chdir $pool_dir;
```

Comment:
Hm, the explanation makes sense how we end up with coverage files referring to differing directories but now it looks like we need to switch back and forth between different directories. Do you think it would be possible to somehow refer to all coverage files by absolute paths instead?
So instead of trying to ensure that we run everything from the os-autoinst root directory do you think it would be possible if we run everything from an external temporary directory so that Devel:Cover would relate to files by absolute paths?

Reply:
I also thought about that but not sure it will help much.

The back and forth is mostly necessary because the tests are working with files created by the isotovideo call (e.g. `autoinst-log.txt`), but they are referring to those files with a relative path.
So the easiest thing for me was to switch back to the pool dir, instead of changing all the file paths in the tests. The latter would be better IMHO, but it is a bit of work.

Reply:
Ok, thanks, I see. Well, we can still look into the alternative later on. The important thing is that you found out the original problem and found a fix